packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -194,6 +194,18 @@ describe('<CurrencySelector/>', () => {
         goToNextStep: jest.fn(),
         resetRealAccountSignupParams: jest.fn(),
         onSubmit: jest.fn(),
```

Comment:
there are no types for props and for constants

Reply:
Will add for props. Constanst, I dont think we need to be explicit